### ISSUE TITLE
bug(ion-datetime) format as MM only

### ISSUE THREAD

**Issue by Newbie012** [ionic-team/ionic#16723]
# Bug Report

**Ionic version:**

[x] **4.x**

**Current behavior:**

If you pick a month in `<ion-datetime displayFormat="MM"></ion-datetime>`, it won't select it. Once you try to select it again, the value will be displayed, but it won't patch the value.

**Expected behavior:**

Choosing a month should be one time, and it should patch the control value.

**Steps to reproduce:**

 - run `<ion-datetime displayFormat="MM"></ion-datetime>`.
 - pick a month.
 - see occurance.

**Related code:**
```
<ion-datetime displayFormat="MM"></ion-datetime>
```

```
insert short code snippets here
```

**Other information:**

**Ionic info:** 

```
insert the output from ionic info here
```

**Comment by liamdebeasi** [comment]
Hi there,

Thanks for opening an issue with us. I am going to close this for now, but we will be re-evaluating the `ion-datetime` functionality and bugs soon during which we will take a closer look at this issue.

Thanks for using Ionic!

**Comment by liamdebeasi** [comment]
Hi there,

This has been fixed via #17706 and will be in the next release of Ionic. I have published a nightly build of Ionic with this fix (`npm install @ionic/core@dev`). Feel free to test it out and provide feedback if you have the time.

Thanks!
